src/bindings/python/tests/mock/ov_mock_py_frontend/mock_py_frontend.hpp

```diff
@@ -429,8 +433,8 @@ struct MOCK_API ModelStat {
     }
 };
 
-class MOCK_API InputModelMockPy : public InputModel {
-    mutable ModelStat m_stat;
+class MOCK_API InputModelMockPy : public IInputModel {
+    static ModelStat m_stat;
```

Comment:
The same

Reply:
Same - now public InputModel is not cast'able to InputModelMockPy, so mock statistics is converted to static, which is enough for these tests